doc/interop-test-descriptions.md

```diff
@@ -1078,6 +1095,15 @@ included in that latency measurement (channel teardown semantics differ widely
 between languages). This latency measurement should also be the value that is
 logged and recorded in the latency histogram.
 
+* `createNewChannel` Function: In channel_soak, the `createNewChannel` function 
+is used by each thread to create a new channel before every RPC. This function 
+ensures that each RPC has a separate channel, preventing race conditions by 
+isolating channels between threads. It shuts down the previous channel (if any)
+and creates a new one for each iteration, ensuring accurate latency measurement
+per RPC.
+* Thread-specific logs will include the thread_id, helping to track performance 
```

Comment:
let's put this bullet point underneath the part that talks about regexes "Upon success" and "Upon failure"